src/service/publicActivity.service.ts

```diff
@@ -292,9 +292,12 @@ export const getSearchActivities = async ({
         return distance <= maxDist;
       });
 
-      return { activities: filteredActivities };
+      return {
+        activities: filteredActivities,
+        total: filteredActivities.length,
+      };
     } else {
-      return { activities };
+      return { activities, total: activities.length };
     }
```

Comment:
use the lib of `paginator`? should have filtered fields of `limit` and `page`?

Reply:
ok, already updated